deps/cares/src/ares_init.c

```diff
@@ -1000,11 +1025,6 @@ static int get_DNS_AdaptersAddresses(char **outptr)
       }
       else if (namesrvr.sa->sa_family == AF_INET6)
       {
-        /* Windows apparently always reports some IPv6 DNS servers that
-         * prefixed with fec0:0:0:ffff. These ususally do not point to
-         * working DNS servers, so we ignore them. */
-        if (strncmp(txtaddr, "fec0:0:0:ffff:", 14) == 0)
-          continue;
```

Comment:
FTR, it's this one: https://github.com/nodejs/node/commit/3afa5e6b714ce3b8bc162f2f186b0690df67edb4 Maybe the comment can be prefixed with "nodejs:" so it's easier to spot that it's a floating patch?